client/plots/singleCellPlot.js

```diff
@@ -82,7 +82,7 @@ class singleCellPlot {
 			if (result == 1 || result == -1) return result
 			else return elem1.sample.localeCompare(elem2.sample)
 		})
-
+		const initialSample = state.config.sample != undefined
```

Comment:
I don't this variable is needed, could just check within the `isVisible()` callback.